release/release.go

```diff
@@ -912,7 +957,116 @@ func uploadRelease(v version.Version) {
 		err = jsonEncoder.Encode(feed)
 		check(err, "encode json feed")
 
-		fmt.Printf("uploading download feed to https://s3.amazonaws.com/downloads.mongodb.org/tools/db/%s\n", feedFilename)
+		log.Printf("uploading download feed to https://s3.amazonaws.com/downloads.mongodb.org/tools/db/%s\n", feedFilename)
 		awsClient.UploadFile("downloads.mongodb.org", "/tools/db", feedFilename)
 	}
 }
+
+var linuxRepoEditions = []string{"org", "enterprise"}
+var linuxRepoVersions = []string{"4.3.0"}
+
+func linuxRelease(v version.Version) {
+	if env.EvgIsPatch() {
+		log.Println("current build is a patch; not performing a linux release")
+		return
+	}
+
+	tasks, err := evergreen.GetTasksForRevision(v.Commit)
+	check(err, "get evergreen tasks")
+
+	distTasks := []evergreen.Task{}
+	for _, task := range tasks {
+		if task.IsPatch() || task.DisplayName != "dist" {
```

Comment:
how does `task.IsPatch()` differ from `env.EvgIsPatch()`? Little confused here because I don't see how one task could be a patch when the whole build is not. Some comments would help clarify.

Reply:
The former checks whether a task fetched from the API is a patch, while the latter checks whether the task in which this code is running is a patch. I've added godoc comments to the relevant functions to clarify.